harness/determined/_trial_controller.py

```diff
@@ -249,82 +249,5 @@ def __init__(self, *args: List[Any], **kwargs: Dict[str, Any]) -> None:
             logging.getLogger().setLevel(log_level)
 
         logging.debug(
-            f"Training coordination initialized on local rank {training_process_rank}, "
-            f"using hvd: {self.hvd_config.use}."
+            f"TrialController initialized on rank {rank}, using hvd: {self.hvd_config.use}."
         )
-
-        # Initialize communication directly between training processes.
-        self.train_process_comm_chief = None  # type: Optional[ipc.ZMQBroadcastServer]
-        self.train_process_comm_worker = None  # type: Optional[ipc.ZMQBroadcastClient]
-        if self.hvd_config.use:
-            self._initialize_train_process_comm()
-
-    def _initialize_train_process_comm(self) -> None:
```

Comment:
:D I actually wrote some similar changes in my `LightningTrial` code. I am so glad to see those changes happen here. Those functions just make no sense to be in the trial controller.

Reply:
Agreed.  (sorry it took me so long to come around, haha)